src/snowflake/snowpark/dataframe.py

```diff
@@ -1373,18 +1373,37 @@ def unpivot(
         return self._with_plan(unpivot_plan)
 
     @df_api_usage
-    def limit(self, n: int) -> "DataFrame":
+    def limit(self, n: int, offset: Optional[int] = 0) -> "DataFrame":
```

Comment:
```suggestion
    def limit(self, n: int, offset: int = 0) -> "DataFrame":
```
Optional means it can be None

Reply:
Right, thank you for catching this!